dahu/core/app/scripts/views/workspace/actions/action.js

```diff
@@ -17,18 +17,121 @@ define([
     /**
      * Generic action view
      */
-    return Marionette.ItemView.extend({ 
+    return Marionette.ItemView.extend({
         
         template: Handlebars.default.compile(actionTemplate),
         
         className: "action",
 
-        events: {
-            "click .actionDelete": "delete"
+        /**
+         * View parameter to switch from extanded view
+         * with full parametrable menu to short header
+         * displaying few read-only information.
+         * True : full menu.
+         * False : short header.
+         * @type {Boolean}
+         */
+        toggle: false,
+
+        /**
+         * Register helpers to be used in the template
+         * when rendering is called
+         * @return {[type]} [description]
+         */
+        templateHelpers: function () {
+            return{
+                isToggled: this.toggle,
+                objects: this.objects,
+                availableTrigger: [
+                    "slide change",
+                    "with previous",
+                    "after previous"
+                ],
+                whichActionPartial: function() {
```

Comment:
maybe the type is not even needed... actually we might not need this method ! just use the same name for the partial: actionParametersPartial